client/src/components/views/Navigation.jsx

```diff
@@ -1,9 +1,52 @@
 import React from 'react'; // React module is imported if you choose to convert to class component, remove the import if not
+import AppBar from '@mui/material/AppBar';
```

Comment:
Not an issue just a suggestion, you can consolidate your imports from mui by pointing to the /material directory and bracketing imports Ex: import { Button, Divider, Toolbar } from '@mui/material';

Reply:
WIll do thanks.